templates/raspberrypi.py

```diff
@@ -122,14 +122,31 @@ def msg_{{key.lower()}}_{{structKey.lower()}}(
     ): # Put params here
         msg = 0
         {% for param_name, param_info in struct.fields|dictsort|reverse %}
-        msg |= {{param_name}} {{"<<{}".format(param_info.offset_in_byte) if param_info.offset_in_byte != 0}} 
+        msg |= {{param_name}} {{"<< {}".format(param_info.offset_in_byte) if param_info.offset_in_byte != 0}} 
         {% endfor %}
         return msg # Return the message data structure here
    
     {% endfor %}
     {% endfor %}
     {% endif %}
 
+    {% if imports %}
```

Comment:
I think we can consolidate this. As we already have for-loops created based on `key,data` and `structKey,struct`, we can put each `decode_` function right after the `msg_`